File: model/src/main/java/com/indracompany/treinamento/model/dto/DepositoDTO.java
package com.indracompany.treinamento.model.dto;

import java.io.Serializable;

import lombok.Data;

@Data
public class DepositoDTO implements Serializable{

	private static final long serialVersionUID = 3143005952249042118L;
	
	private String agencia;
	private String numeroConta;
	private double valor;
	
}

File: view/src/app/core/dtos/deposito.dto.ts
export class DepositoDTO {

    agencia: string;
    numeroConta: string;
    valor: number;

    constructor(obj?) {
        if (obj) {
            this.agencia = obj.agencia;
            this.numeroConta = obj.numeroConta;
            this.valor = obj.valor;
        }
    }
}
